### ISSUE TITLE
[Feature] URI and file acossication

### ISSUE THREAD

**Issue by jkaberg** [hadouken/hadouken#163]
IMO Hadouken shouldn't be modest in its presence on an OS, so how about hooking up the magnet URI and .torrent file with Hadouken? :smile: 

IDK how this works on Linux tho

URI: https://msdn.microsoft.com/en-us/library/aa767914(v=vs.85).aspx (magnet: ...)
File: https://msdn.microsoft.com/en-us/library/windows/desktop/hh127451(v=vs.85).aspx (.torrent)

**Comment by vktr** [comment]
Yes, for a desktop application this is easy to do :) However, since Hadouken may or may not run on a headless server in a wardrobe somewhere, the magnet URI's and file type associations will be difficult to pull off...

Of course - if you run Hadouken on your regular machine, no problem.
